packages/styling/src/interfaces/ITheme.ts

```diff
@@ -16,6 +17,7 @@ export interface ITheme {
    * Customizer will not get the global styles applied to them.
    */
   disableGlobalClassNames: boolean;
+  typography?: ITypography;
```

Comment:
Everything on ITheme should be required by default. The intention is that it's used wherever we generate a theme, so that a compile-time error is thrown if somebody accidentally leaves something out.

Reply:
Can we not use IFontStyles for this?

Reply:
I think we need a more robust interface for generalized typography.